library/src/components/layout/Header/Header.tsx

```diff
@@ -18,21 +28,43 @@ export default function Header(props: headerProps): JSX.Element {
   const handleLogout = () => {
     resetAuth();
   };
-  const languages = useStoreState((state) => state.i18n.languages);
+
+  const theme = useTheme();
+  const desktop = useMediaQuery(theme.breakpoints.up('md'));
 
   return (
-    <StyledMenuContainer>
-      <Breadcrumbs routeMap={routeMap} />
-      <Box>
-        {languages && <LanguageSelector languages={languages} />}
-        <AccountSettings customAvatar={customAvatar}>
-          {children || (
-            <MenuItem key='logout' onClick={handleLogout}>
-              <Typography textAlign='center'>Logout</Typography>
-            </MenuItem>
-          )}
-        </AccountSettings>
+    <Box className='menu-container'>
+      <Box className='start'>
+        {desktop ? (
+          <Breadcrumbs routeMap={routeMap} />
+        ) : (
+          <a className='back-mobile'>
+            <NavigateBeforeRoundedIcon /> Breadcrumb movil
+          </a>
+        )}
```

Comment:
```suggestion
        {desktop && (
          <Breadcrumbs routeMap={routeMap} />
        ) }
        
        {!desktop && (
            <a className='back-mobile'>
                <NavigateBeforeRoundedIcon /> Breadcrumb movil
            </a>
        ) }
```
Would be easier to read